srv/GraspPlanning.srv

```diff
@@ -0,0 +1,27 @@
+# Requests that grasp planning be performed for the target
+# returns a list of grasps to be tested and executed
+
+# the planning group used
+string group_name
+
+# the object to be grasped
+CollisionObject target
+
+# the name that the support surface (e.g. table) has in the collision map
+# can be left empty if no name is available
+string collision_support_surface_name
```

Comment:
At the moment, the MoveGroupInterface only supports setting *one* support surface and the `Pickup` and `Place` actions do too.
If we change this here, we should extend it at all other places in the kinetic branch. (Let's do that in a separate request)
I'm not sure this is useful in practice, but as at least three of you agree, let's change it.